lib/ios/RNNCommandsHandler.m

```diff
@@ -48,6 +49,20 @@ -(instancetype) initWithStore:(RNNStore*)store controllerFactory:(RNNControllerF
 
 -(void) setRoot:(NSDictionary*)layout completion:(RNNTransitionCompletionBlock)completion {
 	[self assertReady];
+
+	if (@available(iOS 9, *)) {
+        if ([_controllerFactory.defaultOptionsDict[@"layout"][@"direction"] isEqualToString:@"rtl"]) {
+            [[RCTI18nUtil sharedInstance] allowRTL:YES];
+            [[RCTI18nUtil sharedInstance] forceRTL:YES];
```

Comment:
Forcing RTL with React native will break Apps that implement RTL first approach, I think the best solution is to leave it to the user to force RTL or not in js.

Reply:
Right, but it's required to correctly apply RTL layout + RTL content.
If We want a RTL layout means we needs RTL content or LTR layout needs LTR content, 
Could you describe some cases that needs RTL layout with LTR content?

Reply:
@omar-dev In new version of RN marginRight is always right, they are introduced  marginStart, marginEnd instead of making marginRight reverse on RTL.